packages/controls/src/controls/combobox/combobox.ts

```diff
@@ -100,7 +106,7 @@ class Definition<C extends Config> extends ControlDefinition<
   }
 
   fromData(data: DataType<C> | undefined): ValueType<C> | undefined {
-    return this.schema.data.optional().parse(data)
```

Comment:
We shouldn't be parsing in `fromData`/`toData`  methods.

Reply:
We have a dedicate method, `safeParse`, for validating that data is in the format that we expect. When the data is coming from the runtime boundary, we (should) run it through `safeParse`. `fromData` is meant for working w/ the data that's already been validated, as reflected by its signature.